cmd/preflight/cmd/check.go

```diff
@@ -21,7 +23,7 @@ func checkCmd() *cobra.Command {
 	_ = viper.BindPFlag("artifacts", checkCmd.PersistentFlags().Lookup("artifacts"))
 
 	checkCmd.AddCommand(checkOperatorCmd())
```

Comment:
Is a similar abstraction needed for checkOperator ?

Reply:
Yes. Separate PR coming.